tgui/packages/tgui-dev-server/link/client.cjs

```diff
@@ -8,112 +8,109 @@ let socket;
 const queue = [];
 const subscribers = [];
 
-const ensureConnection = () => {
-  if (process.env.NODE_ENV !== 'production') {
```

Comment:
Webpack is stupid and won't remove dead code (if(false) block) in production unless it is written exactly this way. I could be wrong, perhaps esbuild now removes it during minification.

Reply:
That's annoying. Is this tgui specific? I'd like to test this locally

Reply:
I tested this, and you are correct on both accounts: Webpack doesn't remove the dead code if there's an early return, however esbuild _does_ with minification

just webpack:
![image](https://github.com/user-attachments/assets/038d4dfd-6632-4017-9a88-a6f684d1dfb6)

with esbuild minifier:
![image](https://github.com/user-attachments/assets/ef396812-a966-43c4-9c07-b3d5b55ac480)

Reply:
marking this as resolved since we're minifying it but will definitely keep it in mind to check in the future